cognite/seismic/protos/v1/seismic_service_messages.proto

```diff
@@ -310,11 +310,15 @@ message TraceBounds {
 message SegYSeismicRequest {
     oneof identifier {
         Identifier seismic = 1;
-        int64 seismic_store_id = 2;
+        int64 seismic_store_id = 2;  // Need to be a data manager or tenant user, not 3rd party, to access by seismic store
     }
+    // This only supports queries to 3D objects.
+    // Deprecated. Use filter instead.
     oneof query {
         Geometry polygon = 3;
```

Comment:
In the previous iteration of this PR, the geometry field was a GeometryFilter message. Now that field's gone entirely. Should it be brought back?

Reply:
(must admit I'm such a lazy f!ck I didn't even go look at the two messages to compare for differences :grimacing: Will go check now =) )

Reply:
`GeometryFilter` adds an interpolation field which as far as I can tell isn't used when streaming raw traces, since we don't evaluate the coordinates of the returned traces before streaming, so I don't think it's needed here

Reply:
Oh right, interpolation method. Hmm. Probably not relevant for SEGY output.